### PR TITLE
add: flowers name translation

### PR DESCRIPTION
**Pull request by elenamkoian**

### CODE REVIEW

File: src/locale/catalogue.tsx [hunk 1]
```diff
@@ -51,13 +51,13 @@ const CataloguePageTranslation = {
...
-        cream: "Kремовый",
-        yellow: "Желтый",
-        orange: "апельсиновый",
-        red: "Красный",
-        purple: "Розовый",
-        bicolor: "Двухцветный",
+        White: "Белый",
+        Cream: "Kремовый",
+        Yellow: "Желтый",
+        Orange: "апельсиновый",
```
Comment (menuason) [inline src/locale/catalogue.tsx]: Capital

File: src/pages/Catalogue/CatalogueFilter.tsx [hunk 2]
```diff
@@ -51,7 +52,8 @@ const CatalogueFilter: FC<CatalogueFilterProps> = ({
           }`}
         >
           <FilterItem
-            items={colors}
+            colorItems={colors}
+            // items={colors}
```
Comment (menuason) [inline src/pages/Catalogue/CatalogueFilter.tsx]: is it right here ? comment

File: src/pages/Catalogue/FilterItem.tsx [hunk 3]
```diff
@@ -1,22 +1,47 @@
 import { FC } from "react";
 import { CheckboxBloom } from "../../components/Checkbox";
+import { Colors } from "../../components/FilterSection";
+import { useTranslation } from "react-i18next";
```
Comment (menuason) [inline src/pages/Catalogue/FilterItem.tsx]: pls, up it import

**Review by menuason — APPROVED** [review]

File: src/locale/catalogue.tsx (hunk 1 above)
Comment (elenamkoian) [reply src/locale/catalogue.tsx]: it should be that way, as color names in our catalogueFlowers array are capitalized

**Pull request merged**